crates/webidl/src/main.rs

```diff
@@ -0,0 +1,114 @@
+use anyhow::{Context, Result};
+use sourcefile::SourceFile;
+use std::ffi::OsStr;
+use std::fs;
+use std::path;
+use std::process::Command;
+
+fn unwrap_not_found(err: std::io::Result<()>) -> std::io::Result<()> {
```

Comment:
I think it's fine to basically delete this method and just check if something exists before we try to remove it to avoid having to handle special errors here.

Reply:
That is susceptible to race conditions, but I guess it doesn't matter much in this case.